src/app/(authenticated)/monitoring/components/monitoring-presence-client.tsx

```
@@ -170,6 +234,48 @@ function RombelCard({ rombel }: { rombel: RombelWithPresence }) {
                 </Badge>
               </div>
             </div>
+
+            <Dialog open={isDialogOpen} onOpenChange={setIsDialogOpen}>
+              <DialogContent className="sm:max-w-lg">
+                <DialogHeader>
+                  <DialogTitle>
+                    {statusLabel[(selectedStatus || "h") as "h"]} - {rombel.nama}
+                  </DialogTitle>
+                  <DialogDescription>
+                    Daftar siswa dengan status "{selectedStatus ? statusLabel[selectedStatus] : ""}" pada sesi ini.
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)